fastlane/report.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should definitely be in .gitignore.

Reply:
Hello @mkj-is 
I did not know how reports are important for you. If you want I can add it to the .gitignore.

Reply:
@mkj-is Done.